.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: node_js
-cache: yarn
 notifications:
   email: false
 node_js:
   - "4"
+  - "5"
   - "6"
+  - "7"
   - "8"
+install:
+  - npm i -g npm
```

Comment:
You shouldn't do this for all node versions; you should do this only where is needed, probably 4 and 6 like this

```yml
before_install:
  - if [[ `npm -v` != 5* ]]; then npm install -g npm@5; fi
```